core/domain/suggestion_registry_test.py

```diff
@@ -114,6 +161,362 @@ def test_get_score_part_helper_methods(self):
         self.assertEqual(suggestion.get_score_type(), 'content')
         self.assertEqual(suggestion.get_score_sub_type(), 'Algebra')
 
+    def test_validate_suggestion_type(self):
```

Comment:
In all of these tests (for validate()), can we first do a successful validation followed by a broken one (in the same test, basically called suggestion.validate() before you change it)? So that we know that the field you change is the one causing the failure?

Reply:
Done